layouts/partials/fragments/items.html

```diff
@@ -1,4 +1,5 @@
 {{- $items := .resources.Match (printf "%s/*" .Name) -}}
+{{- $path := .Name -}}
```

Comment:
Isn't there any way to avoid doing this? It's a bit annoying.

Reply:
hehe yeah i didn't want to bring that back seeing it was already removed but i needed a way to get the filename